src/components/layout/AppBar.js

```diff
@@ -415,10 +415,7 @@ function DEAppBar(props) {
                                 profile={userProfile}
                                 onLogoutClick={onLogoutClick}
                                 onManageAccountClick={() =>
-                                    window.open(
-                                        constants.CYVERSE_USER_PORTAL,
-                                        "_blank"
-                                    )
+                                    window.open(config?.userPortalURL, "_blank")
```

Comment:
Not sure what this would do if config is null -- I think it'd call `window.open(null, "_blank")`, which testing in my browser appears to try to open that as though null is a string and a relative path. Which is probably not what we want. Not sure what makes sense to put here instead -- possibly just make it no-op if `config` is null?

Reply:
This was an odd case. I wasn't expecting `config` ever to be null, but when the page was rendered the first time, it was always null, then it was immediately re-rendered with the configuration values available. Several other files simply return null if the configuration value is missing in similar cases, though. My hope was that a redirect to null would never come up in production, but there's probably a better way to accomplish that.